packages/checkout/utils/validation/index.ts

```diff
@@ -1,58 +1,2 @@
-/**
- * External dependencies
- */
-import { __, sprintf } from '@wordpress/i18n';
-
-/**
- * Ensures that a given value contains a string, or throws an error.
- */
-export const mustContain = (
-	value: string,
-	requiredValue: string
-): true | never => {
-	if ( ! value.includes( requiredValue ) ) {
-		throw Error(
-			sprintf(
-				/* translators: %1$s value passed to filter, %2$s : value that must be included. */
-				__(
-					'Returned value must include %1$s, you passed "%2$s"',
-					'woo-gutenberg-products-block'
-				),
-				requiredValue,
-				value
-			)
-		);
-	}
-	return true;
-};
-
-/**
- * Converts an input's validityState to a string to display on the frontend.
- *
- * This returns custom messages for invalid/required fields. Other error types use defaults from the browser (these
- * could be implemented in the future but are not currently used by the block checkout).
- */
-export const getValidityMessageForInput = (
```

Comment:
We've removed the function here, but it is still used in different text-input components and validation tests. 

- https://github.com/woocommerce/woocommerce-blocks/blob/add/7722-add-client-side-postcode-validation-npm/packages/checkout/components/text-input/validated-text-input.tsx#L24
- https://github.com/woocommerce/woocommerce-blocks/blob/add/7722-add-client-side-postcode-validation-npm/packages/checkout/utils/validation/test/index.tsx#L20

Reply:
This change is not related to this PR, isn't it, @tarunvijwani?

Reply:
Not sure which PR was responsible for it. I noticed it in this PR file changes. Thank you, I noticed you fixed it https://github.com/woocommerce/woocommerce-blocks/pull/7945/commits/ef2a3e952d29e9cde8869c39381edb837f94aa48 commit. 👍